sql/migrations/schema-postgresql-move-to-mirrors.sql

```diff
@@ -0,0 +1,319 @@
+DROP TABLE IF EXISTS filemetadata, mirrors CASCADE;
+DROP MATERIALIZED VIEW IF EXISTS filemetadata_mirror_count;
+
+CREATE TABLE filemetadata
+(
+    id integer GENERATED ALWAYS AS IDENTITY,
+    path character varying NOT NULL,
+    path_hash bytea GENERATED ALWAYS AS (digest((path)::text, 'sha256'::text)) STORED,
+    mtime timestamp with time zone,
+    size bigint,
+    md5 bytea,
+    sha1 bytea,
+    sha256 bytea,
+    sha1piecesize integer,
+    sha1pieces bytea,
+    btih bytea,
+    pgp text,
+    zblocksize smallint,
```

Comment:
We should use INT instead of SMALLINT, which is mileted to 32K : zsync algorithm doesn't have practical limit for block size and e.g. block size of 1M is much suitable for huge files (see #47 )